core/src/main/java/org/mobicents/ha/javax/sip/LoadBalancerHeartBeatingServiceImpl.java

```diff
@@ -639,7 +639,7 @@ protected void updateConnectorsAsSIPNode() {
 						if(jvmRoute != null) node.getProperties().put("jvmRoute", jvmRoute);
 						
 						node.getProperties().put("version", System.getProperty("org.mobicents.server.version", "0"));
-						
+						node.getProperties().put("sessionId", ""+System.currentTimeMillis());
 						if(gracefullyShuttingDown) {
```

Comment:
@deruelle yes, I made test: https://github.com/RestComm/load-balancer/blob/master/jar/src/test/java/org/mobicents/tools/sip/balancer/operation/MediaFailureTest.java. I guessed that adding and removing connectors it is not rarely operation, so we may skip this checking.